app/models/exchange_rate_currency_rate.rb

```diff
@@ -111,6 +111,12 @@ def by_month_and_year(month, year)
       )
       .order(Sequel.desc(:applicable_date))
     end
+
+    # This query removes country/currency pairings where the country/currency
+    # is expiring before the end of the rate
+    def without_expired_countries
```

Comment:
This is added because if a country has expired in Oct like Liberia, we dont want to include it if the exchange rate expires after this date.

Reply:
Add specs to this file after chat with @jebw